src/sentry/utils/sdk.py

```diff
@@ -90,6 +90,10 @@ def configure_sdk():
         upstream_transport = make_transport(get_options(options))
 
     def capture_event(event):
+        if event.get('type') == 'transaction':
+            metrics.incr('internal.apm.events', skip_internal=False)
```

Comment:
Why is this doing `skip_internal=False`? We don’t do this on any new metrics. It shouldn’t be used for anything really without a very strong argument otherwise.